src/index.js

```diff
@@ -141,7 +141,7 @@ const App = () => {
                 </thead>  
 
                 <tbody>  
-                    <ToDoItems todos={sortedTodos} toggleCompleted={toggleCompleted} />
+                    <ToDoItems todos={sortedTodos} toggleCompleteItem={toggleCompleteItem} />
```

Comment:
So this could be 
```js
<ToDoItems todos={sortedTodos} onComplete={handleComplete} />
// OR
<ToDoItems todos={sortedTodos} onItemComplete={handleItemComplete} />
// OR
<ToDoItems todos={sortedTodos} onCompleteItem={handleCompleteItem} />
```
Or anything you think make sense.